### PR TITLE
Custom layout

### PR DESCRIPTION
**Pull request by ehildenb**
More appropriate name for #52 .

### CODE REVIEW

File: API/pom.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright (c) 2015 K Team. All Rights Reserved. -->
+<!-- Copyright (c) 2018 K Team. All Rights Reserved. -->
```
Comment (dwightguth) [inline API/pom.xml]: these should become 2015-2018, etc throughout

**Review by dwightguth — CHANGES_REQUESTED** [review]
I have one request, but you should be able to implement it procedurally.

**Comment by ehildenb** [comment]
@dwightguth I think I got all of the copyrights correct this time. I also ran `dos2unix` on all files in the repo, there were some with funky line-endings.

**Review by dwightguth — APPROVED** [review]
this looks really good and is an awesome new feature we've wanted for a while, thanks!

**Pull request merged**